website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md

```diff
@@ -208,6 +208,67 @@ the integration between Okta and dbt Cloud.
 3. After creating the Okta application, follow the instructions in the [dbt Cloud Setup](#dbt-cloud-setup)
 section to complete the integration.
 
+## Google integration
+
+_Use this section if you are configuring Google as your identity provider_.
+
+### Configure the Google application
+
+Sign into your **Google Admin Console** via an account with super administrator privileges.
+
+* From the Admin console Home page, go to **Apps** > **Web and mobile apps**.
```

Comment:
I'm not sure how `<` or `>` are read by screen readers so I tend to use "and then click" just in case.

Reply:
oh good tip! thank you!